internal/pkg/service/buffer/api/receive/importer.go

```diff
@@ -60,11 +70,35 @@ func NewImporter(d dependencies) *Importer {
 		watcher:        d.WatcherAPINode(),
 		statsCollector: d.StatsCollector(),
 		quota:          quota.New(d),
+		metrics: metrics{
+			Count:    d.Telemetry().Meter().Counter("keboola.go.buffer.ingress_records", "Count of received records.", ""),
+			BodySize: d.Telemetry().Meter().Counter("keboola.go.buffer.ingress_bytes", "Sum of all request body bytes.", "bytes"),
+		},
 	}
 }
 
 // CreateRecord in etcd temporal database.
-func (i *Importer) CreateRecord(ctx context.Context, d requestDeps, receiverKey key.ReceiverKey, secret string, bodyReader io.ReadCloser) error {
+func (i *Importer) CreateRecord(ctx context.Context, d requestDeps, receiverKey key.ReceiverKey, secret string, bodyReader io.ReadCloser) (err error) {
+	// Metrics
+	var bodySize int64
+	defer func() {
+		attrs := []attribute.KeyValue{
+			attribute.String("projectId", receiverKey.ProjectID.String()),
+			attribute.String("receiverId", receiverKey.ReceiverID.String()),
+			attribute.String("source", "http"),
+			attribute.Bool("is_success", err == nil),
+		}
```

Comment:
No pri tejto metrike je naviac `projectId` a `receiverId`.

Spravil som to tak, pretoze v DD sa plati za kazdu unikatnu kombinaciu metrik.
Teda chceme mat nejaku metriku, ktora ma minimalny pocet atributov, ale ma atribut `projectId`.
Diskutovane s Kacurezom tu: https://keboolaglobal.slack.com/archives/C055WKPHYSD/p1689144823322729?thread_ts=1684394765.432299&cid=C055WKPHYSD